PubNub/Data/PNAES.m

```diff
@@ -356,8 +359,12 @@ + (NSError *)errorFor:(CCCryptorStatus)status {
             errorCode = kPNAESInsufficientMemoryError;
             break;
         case kCCDecodeError:
+#ifdef PGDROID
+#   warning HC SVNT DRACONES! : CommonCrypto is not fully implemented on PGDroid ...
```

Comment:
In particular `kCCOverflow` and `kCCRNGFailure` do not exist in platform.  The ideal fix will be to bring in the new Apple headers (into Apportable) and iterate on supporting them.  This is likely to be a large platform task.